html/docs/include/settings.php

```diff
@@ -568,6 +568,35 @@
 		FUNC_PROFILER
 	),
 
+    'gc_stats_enable' => array(
+        'bool', 'false', null,
```

Comment:
the null should be ``">= 2.6"`` to show in which version this was introduced (3 times).